salt/states/zenoss.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+'''
+State to manage monitoring in Zenoss.
+
```

Comment:
@iamseth, 2015.8 (previously Beryllium) has already been forked, so this will go into the next release, Boron.